plugins/builds/index.js

```diff
@@ -367,15 +367,15 @@ module.exports = {
       }
     };
     appkit.args
-      .command('builds', 'list available builds', require_app_option, list.bind(null, appkit))
-      .command('builds:create', 'create a new build', create_build_option, create.bind(null, appkit))
-      .command('builds:info [ID]', 'view build info, ID can be "latest" for latest build.',  require_app_option, info.bind(null, appkit))
-      .command('builds:output [ID]', 'view output for a build, ID can be "latest" for latest build.', require_logs_option, logs.bind(null, appkit))
-      .command('builds:auto', 'set the auto deploy hooks for your source control', require_auto_build_option, auto.bind(null, appkit))
-      .command('builds:auto:info', 'get the auto build/deploy information for this app.', require_app_option, info_auto.bind(null, appkit))
-      .command('builds:auto:remove', 'remove the auto build (if set)', require_app_option, remove_auto.bind(null, appkit))
-      .command('builds:rebuild ID', 'rebuild a previous build.', require_app_build_option, rebuild.bind(null, appkit))
-      .command('builds:stop ID', 'stop a running build', require_app_build_option, stop.bind(null, appkit))
+      .command('builds', false, /*'list available builds',*/ require_app_option, list.bind(null, appkit))
```

Comment:
Looks like setting false as a description disables the command? Why do this instead of removing the lines?

Reply:
Setting it to false hides it in the CLI, so that its still available (and backwards compatible), we haven't really changed anything on the backing API's this is more or less jsut a UX change.